doc/src/arch/reference.rst

```diff
@@ -990,6 +995,7 @@ They describe how a complex block interfaces with the inter-block world.
 
     When generating the FPGA routing architecture VPR will try to make 'good' choices about how pins and wires interconnect; for more details on the criteria and methods used see :cite:`betz_automatic_generation_of_fpga_routing`.
 
+    If ``<fc>`` is not specified for a complex block, the values from the architecture's ``<default_fc>`` tag will be used if available.
```

Comment:
Its probably best to make this more prominent (perhaps as a `.. note::`, like in `<switchblock_locations>`).